### PR TITLE
Add furnace data for 90.1 PRM

### PR DESCRIPTION
**Pull request by AmmarDeh**

### CODE REVIEW

File: building_energy_standards_data/database_files/hvac_minimum_requirement_furnaces_90_1_prm.json
Comment (lymereJ) [inline building_energy_standards_data/database_files/hvac_minimum_requirement_furnaces_90_1_prm.json:661]: This is indeed correct, but I wonder why he leave the choice to the user. I'm checking to see if they are equivalent.

**Review by lymereJ — APPROVED** [review]
Thanks, @AmmarDeh! Looks great.

Some notes for @lzwang26 on some of the changes that might impact the IECC data:
- 90.1-2022 includes requirements for standby and off mode power for boilers for residential applications (small capacities); These fields were added to the tables and the missing residential boiler requirements were added. I suspect that these should also be in IECC
- Use of `Electricity` instead of `Electric` as the furnace `fuel_type` fields to be consistent with other tables.
- The `product_class` field/column name for water heaters was changed to `equipment_type` to be consistent with other equipment tables

**Comment by lymereJ** [comment]
I'll wait for @lzwang26 review before merging but I think this is ready.

**Review by lzwang26 — APPROVED** [review]
Looks good to me. Ready to merge.

File: building_energy_standards_data/database_files/hvac_minimum_requirement_furnaces_90_1_prm.json
Comment (AmmarDeh) [reply building_energy_standards_data/database_files/hvac_minimum_requirement_furnaces_90_1_prm.json:661]: This is one of the footnotes related to this from 90.1-2016:
b. Combination units not covered by the U.S. Department of Energy Code of Federal Regulations 10 CFR 430 (three-phase power or cooling capacity greater than or equal to 65,000 Btu/h) may comply with either rating.

**Pull request merged**
